manual/development-process/production-deployment.md

```diff
@@ -0,0 +1,175 @@
+production multiple times a day if necessary. The [end-to-end tests] which run on integration and staging help us catch
+regression before code is deployed to production. Deploying frequent changes to production means that each deployment is
+small, which makes deployments less risky and easier to fix.
+
+TDR is made up of [multiple services][tdr-architecture], each of which are deployed separately. Infrastructure is also
+defined in Terraform repos, and is deployed to each environment in the same way way.
+
+## When to deploy
+
+You should deploy to Staging and Production as soon as possible after merging the change and checking it on Integration.
+This means that you can deploy your own change. If you leave your change undeployed and someone else deploys it at the
+same time as theirs, and failures it will be harder for them to debug.
+
+We have agreed to some rules about when it is OK to deploy to Production:
+
+- Only when you have time to monitor the deployment. Decide how long you need based on the size and complexity of the
+  change.
+- Only when there are other people around. We are a small team with flexible working hours, so there are no restrictions
+  based on the time of day, but you should make sure that at least one other developer is around to help debug any
+  deployment issues.
+- Not on a Friday or the day before a bank holiday. Ideally, we would also deploy on Fridays to avoid a build-up of
+  changes that would make the deployment larger, but we need to demonstrate that our deployments are low-risk and that
+  we can catch issues early with tests and monitoring.
+
+These rules only apply to Production, but it's usually better to apply them to Staging as well. This means that you can
+Production shortly after deploying to Staging, which prevents coordination problems where changes are deployed to
+Production in the wrong order.
+
+## How to deploy a change to production
+
+The developer who made a change is responsible for the deployment. If you do not have permission to deploy to staging
+and production, you should ask a developer with production access to run the actual deployments, but you yourself should
+still follow the steps below to make sure everything is working at each step.
+
+This guide explains how to deploy a single service. If you need to deploy multiple changes (e.g. a database, API and
+frontend change), you should follow these steps separately for each service. Each step needs to be compatible with the
+other services currently deployed to each environment. For example, you should not deploy an API change which depends on
+a database change that hasn't been deployed yet. The end-to-end tests should help you catch problems like this so that
+you can roll out the change without any downtime.
+
+**Always deploy to each environment in order.** Staging should never be ahead of Integration, and Production should
```

Comment:
Could potentially reword this: Deployment to Integration should be first, followed by Staging, and finally Production.

Reply:
Done. Maybe "ahead of" is a bit jargony. 🙂